package.json

```diff
@@ -8,17 +8,12 @@
     "test": "tests"
   },
   "scripts": {
-    "server": "npm run enforce-engine-versions && NODE_PATH=$NODE_PATH:. node --optimize_for_size app/server.js",
+    "server": "NODE_PATH=$NODE_PATH:. node --optimize_for_size app/server.js",
     "start": "ember server",
-    "build-dev": "npm run enforce-engine-versions && ember build",
-    "build": "npm run enforce-engine-versions && ember build --environment production",
-    "enforce-engine-versions": "node ./scripts/enforce-engine-versions.js",
-    "preinstall": "npm run enforce-engine-versions",
-    "pretest": "npm run enforce-engine-versions",
-    "prestart": "npm run enforce-engine-versions"
+    "build-dev": "ember build",
+    "build": "ember build --environment production"
   },
   "repository": "https://github.com/mirego/ember-boilerplate",
-  "engine-strict": true,
   "engines": {
     "node": "~14.15",
```

Comment:
I was wondering if that was still useful, after seeing [this](https://github.com/mirego/ember-boilerplate/actions/runs/386072776), I can infer that it's used for some linting rules!